datus/cli/interactive_init.py

```diff
@@ -226,14 +230,26 @@ def _configure_llm(self) -> bool:
         success, error_msg = self._test_llm_connectivity()
         if success:
             console.print("✔ LLM model test successful\n")
+            self._can_init_metrics = providers[provider]["type"] in ("deepseek", "claude")
             return True
         else:
             console.print(f"❌ LLM connectivity test failed: {error_msg}\n")
+            self._can_init_metrics = False
             return False
 
     def _configure_namespace(self) -> bool:
         """Step 2: Configure namespace and database."""
         console.print("[bold yellow][2/5] Configure Namespace[/bold yellow]")
+        config_choice = Prompt.ask(
```

Comment:
Don't add these logic in init. We should add another document to tell customers the detail of california schools dataset.